### DISCUSSION TITLE
Kolonka „Místo“ u akce

### DISCUSSION THREAD

**Discussion by Grejdy** [bosancz/bosan.cz#591] [Ideas]
Když si člověk vytváří akci, má v interní sekci mimo jiné i kolonku „Místo“. Tahle kolonka však nemá z pohledu veřejné sekce žádný význam (kontroloval jsem to, místo srazu se píše přímo do kolonky „Sraz“ - předpokládám mimo jiné proto, aby mohl být konec jinde - třeba na jiném nádraží). Jediné místo, kde je obsah kolonky „Místo“ vidět, je vedle datumu v náhledu akce v interní sekci.

Tím pádem mi tato kolonka přijde aktuálně zbytečná a akorát vedoucí potenciálně může plést. Mám dva návrhy, co s tím dělat:

1. udělat z toho reálnou informaci pro účastníky, která bude vidět i ve veřejné sekci (aby dítě/rodič už předem viděl/o třeba, že se jede na řeku x/y či do toho a toho pohoří a přizpůsobit tomu svou účast a sbalené vybavení)
2. vyhodit z webu tuhle informaci úplně

Jak to vidíte vy?

**Comment by elenfant** [comment]
Nekontroloval jsem to, ale tipoval bych,  že to bude údaj, který se použije v ohlášce a/nebo vyúčtování.

Bod 1. to nicméně nevylučuje (obvykle se asi tahle informace nachází až v samotné přihlášce).

**Reply by Grejdy** [reply]
Vidíš, to mne nenapadlo (a samozřejmě máš pravdu, že se to propisuje do ohlášky a účtování). Tím pádem možnost 2. asi vypadává (ledaže bysme naznali, že to pro nás není hodnotná informace, ale to nepředpokládám).
Naopak 1. možnost by tím pádem mohla mít ještě jeden přínos, a sice že by se mohl zvýšit počet případů, kdy tahle informace bude existovat (respektive by se tím snížil počet lidí, co se na tuhle informaci vykašlou, což se sice, co jsem tak zkoumal, moc neděje, ale bude to spíš tím, že „vidím kolonku, tak jí vyplním a neřeším proč“, než že by si dávali pozor na správně vyplněné kolonky v ohlášce).

**Comment by Grejdy** [comment]
A současně by možná stálo za to trošku upravit popis/umístění té kolonky v interní, aby si to lidi nepletli právě s místem srazu (jakože z poslední doby minimálně o jednom takovém případu vím).
